sa/model.go

```diff
@@ -197,18 +202,17 @@ type challModel struct {
 	Error  []byte          `db:"error"`
 	// This field is unused, but is kept temporarily to avoid a database migration.
 	// TODO(#1818): remove
-	Validated        *time.Time `db:"validated"`
```

Comment:
@rolandshoemaker I think this is the only unresolved piece of feedback from @jsha. I approved the PR but wanted to make sure this wasn't missed.